lightweight-accounts/src/extensions.ts

```diff
@@ -1,10 +1,9 @@
 import { computed } from 'vue'
 import { useGettext } from 'vue3-gettext'
-import { ApplicationSetupOptions, Extension, useStore } from '@ownclouders/web-pkg'
-import { User } from '@ownclouders/web-client'
+import { ApplicationSetupOptions, SidebarNavExtension, useUserStore } from '@ownclouders/web-pkg'
```

Comment:
Pending test, as it requires fixes in backend. So we can merge and then get back to it.